core/domain/question_domain.py

```diff
@@ -196,6 +196,18 @@ class MigrateStateSchemaToLatestVersion(QuestionChange):
     to_version: str
 
 
+class QuestionSuggestionChangeDict(TypedDict):
+    """Dictionary representing the QuestionSuggestionChange domain object."""
+
+    id: None
```

Comment:
Why `None`?

Reply:
Basically, whenever we are getting question suggestion dict from the front end we are not getting `id` with it. Can you please check this comment https://github.com/oppia/oppia/pull/16532#discussion_r1020978242

Reply:
> should we have an explanatory comment why is it None

Done